src/content/docs/validator/amplifier/verifier-onboarding.mdx

```diff
@@ -103,6 +103,18 @@ Note: If you are running a validator as well, you should use a different node fo
 Note: You also must run different `tofnd` instances for `vald` (for your validator) and `ampd` (for your verifier). You cannot use the same `tofnd` instance for both.
 </Callout>
 
+`tofnd` configuration requires the following configuration. These control timeout configuration for communication with tendermint gRPC and tofnd.
+
+```toml
+[tm_grpc_timeout]
+nanos="0"
+secs="5"
+
+[tofnd_config.timeout]
+nanos="0"
+secs="3"
+```
+
```

Comment:
this is part of the ampd config file, so just add it to the section further down that has the full ampd config.